cityofaustin/atd-moped
# Refactor Funding table columns and autocompletes

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/18669

I reorganized some of the files, made a folder to keep the Funding Table components, and moved the autocomplete components out of the funding table file. I also memoized the columns and fixed the autofocus when adding a new line to the table. 

## Testing
**URL to test:** 

https://deploy-preview-1395--atd-moped-main.netlify.app/moped/projects/2035?tab=funding


**Steps to test:**

Add a new funding source, note that the source field autofocuses. Test out the autocomplete inputs, edit an existing row, etc. Kick the moped tires. 

---
#### Ship list
~- [ ] Product manager added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable~
